src/services/pull/usePrefetchSingleFileComp.tsx

```diff
@@ -0,0 +1,226 @@
+import { useQueryClient } from '@tanstack/react-query'
+import { z } from 'zod'
+
+import {
+  FirstPullRequestSchema,
+  MissingBaseCommitSchema,
+  MissingBaseReportSchema,
+  MissingComparisonSchema,
+  MissingHeadCommitSchema,
+  MissingHeadReportSchema,
+} from 'services/comparison'
+import {
+  RepoNotFoundErrorSchema,
+  RepoOwnerNotActivatedErrorSchema,
+} from 'services/repo'
+import Api from 'shared/api'
+import { NetworkErrorObject } from 'shared/api/helpers'
+import A from 'ui/A'
+
+import { FileComparisonWithBase } from './fragments'
+import { transformImpactedFileData } from './utils'
+
+const CoverageLineSchema = z.enum(['H', 'M', 'P'])
+
+const ComparisonSchema = z.object({
```

Comment:
These may be similar but they actually use two different fragments so changing to use this schema could cause issues if the fragment that is unrelated is changed.

Reply:
I will move it to the fragments directory so that it lives next to the fragment that it belongs with, so when the other part of the hook is updated they can re-use the schema.